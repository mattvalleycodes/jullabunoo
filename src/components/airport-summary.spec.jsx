import * as React from "react";
import renderer from "react-test-renderer";

import AirportSummary from "./airport-summary";

describe("<AirportSummary /> component", () => {
  test("renders the airport information", () => {
    const airport = renderer
      .create(
        <AirportSummary
          code="BEK"
          name="Rae Bareli"
          city="Rae Bareli City"
          country="India"
          region="Asia"
        />,
      )
      .toJSON();

    expect(airport).toMatchSnapshot();
  });
});
