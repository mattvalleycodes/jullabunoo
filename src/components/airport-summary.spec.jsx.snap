// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AirportSummary /> component renders the airport information 1`] = `
<a
  className="airport-summary"
  href="/airport/BEK"
  onClick={[Function]}
>
  <h1
    className="airport-summary__name"
  >
    Rae Bareli
  </h1>
  <span
    className="airport-summary__code"
  >
    BEK
  </span>
  <span
    className="airport-summary__location"
  >
    Rae Bareli City
    , 
    India
     -- 
    Asia
  </span>
</a>
`;
